src/widget/form/profileform.cpp

```diff
@@ -361,7 +361,12 @@ void ProfileForm::onDeleteClicked()
                 tr("Are you sure you want to delete this profile?", "deletion confirmation text")))
     {
         Nexus& nexus = Nexus::getInstance();
-        nexus.getProfile()->remove();
+
+        if(!nexus.getProfile()->remove())
+        {
+            GUI::showError(tr("Files could not be deleted!"), tr("Some files could not be deleted, please manually remove them."));
```

Comment:
> Should the prompt show the names of the file that could not be deleted?

Maybe you could list all filenames that couldn't be deleted at once? Might produce a space problem in the error message tough...